docs/MapColonies/Raster/services/catalog/README.md

```diff
@@ -12,7 +12,7 @@ tags:
 ---
 
 # Catalog Service 🗃️
- [Catalog Service for the Web (CSW)](/docs/ogc/protocols/ogc-csw) is a standard for exposing a catalog of geospatial records in XML on the Internet (over HTTP). The catalog is made up of records that describe geospatial data, geospatial services (e.g. [WMTS](/docs/ogc/protocols/ogc-wmts)), and related resources.
+ [Catalog Service for the Web (CSW)](/docs/ogc/protocols/ogc-csw/ogc-csw) is a standard for exposing a catalog of geospatial records in XML on the Internet (over HTTP). The catalog is made up of records that describe geospatial data, geospatial services (e.g. [WMTS](/docs/ogc/protocols/ogc-wmts)), and related resources.
```

Comment:
![image](https://github.com/MapColonies/developer-portal/assets/18442671/c1eaa373-f20f-4b2e-bb69-fd2aa01230f6)
why this property has a different color from all of the rest?

Reply:
Fixed.